specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json

```diff
@@ -1241,6 +1241,54 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/locations/{locationName}/namespaceOperations/{operationResultId}": {
```

Comment:
Same comment as in https://github.com/Azure/azure-rest-api-specs/pull/8110

You can make both of these changes in one PR. This would allow us to have one conversation/approval across multiple teams instead of two.